curriculum/challenges/english/25-front-end-development/quiz-css-flexbox/66ed8fe7f45ce3ece4053eb2.md

```diff
@@ -514,3 +514,445 @@ Dynamically adjusting distribution and alignment of items for different viewport
 
 Fixed sizing and space between items like a grid.
 
+## --quiz--
+
+### --question--
+
+#### --text--
+
+What is the default value of the `flex-direction` property?
+
+#### --distractors--
+
+`column`
+
+---
+
+`row-reverse`
+
+---
+
+`column-reverse`
+
+#### --answer--
+
+`row`
+
+### --question--
+
+#### --text--
+
+Which property defines the alignment along the cross axis?
+
+#### --distractors--
+
+`justify-content`
+
+---
+
+`flex-wrap`
+
+---
+
+`flex-direction`
+
+#### --answer--
+
+`align-items`
+
+### --question--
+
+#### --text--
+
+What value of `align-items` aligns items to the baseline of the container?
+
+#### --distractors--
+
+`flex-start`
+
+---
+
+`flex-end`
+
+---
+
+`center`
+
+#### --answer--
+
+`baseline`
+
+### --question--
+
+#### --text--
+
+What does the `flex-grow` property control?
+
+#### --distractors--
+
+The alignment of items along the cross axis.
+
+---
+
+The wrapping behavior of flex items.
+
+---
+
+The spacing between flex items.
+
+#### --answer--
+
+The ability of a flex item to grow relative to others.
+
+### --question--
+
+#### --text--
+
+What is the initial value of the `flex-wrap` property?
+
+#### --distractors--
+
+`wrap`
+
+---
+
+`wrap-reverse`
+
+---
+
+`inherit`
+
+#### --answer--
+
+`nowrap`
+
+### --question--
+
+#### --text--
+
+Which property is used to define the space between flex items?
+
+#### --distractors--
+
+`align-items`
+
+---
+
+`flex-wrap`
+
+---
+
+`flex-direction`
+
+#### --answer--
+
+`gap`
+
+### --question--
+
+#### --text--
+
+What value of `justify-content` aligns items to the start of the main axis?
+
+#### --distractors--
+
+`center`
+
+---
+
+`space-between`
+
+---
+
+`space-around`
+
+#### --answer--
+
+`flex-start`
+
+### --question--
+
+#### --text--
+
+What value of `align-content` spaces items evenly with equal space around them?
+
+#### --distractors--
+
+`space-between`
+
+---
+
+`flex-start`
+
+---
+
+`center`
+
+#### --answer--
+
+`space-around`
+
+### --question--
+
+#### --text--
+
+What does the `order` property do in a flex container?
+
+#### --distractors--
+
+Defines the alignment of items along the cross axis.
+
+---
+
+Controls the wrapping behavior of flex items.
+
+---
+
+Sets the spacing between flex items.
+
+#### --answer--
+
+Specifies the order of the flex items.
+
+### --question--
+
+#### --text--
+
+Which property allows flex items to shrink if necessary?
+
+#### --distractors--
+
+`flex-grow`
+
+---
+
+`flex-wrap`
+
+---
+
+`align-items`
+
+#### --answer--
+
+`flex-shrink`
```

Comment:
same issue here

was this covered in the lectures or workshop?

If this was covered please provide the link in the course where it was covered in your reply and we can keep this question. Otherwise, this needs to be replaced